<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Markers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="3">
       <widget class="QLineEdit" name="eAnswerCorrect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="hasQuestionFinish">
        <property name="text">
         <string>Question finish marker</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="eCorrect">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Question cloze marker</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="eCloze">
        <property name="text">
         <string>$</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Correct marker</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Section marker</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="hasAnswerCorrect">
        <property name="text">
         <string>Correct answer</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="eQuestionFinish">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&gt;?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="eSubsection">
        <property name="text">
         <string>@</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Question marker</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="hasQuestionStart">
        <property name="text">
         <string>Question start marker</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="eQuestionStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>?&lt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="eAnswerIncorrect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Subsection marker</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="hasAnswerIncorrect">
        <property name="text">
         <string>Incorrect answer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="eQuestion">
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="eSection">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="eNoShuffle">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>!</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="hasNoShuffle">
        <property name="text">
         <string>Disable unsort</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="eMatch">
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Match marker</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hasAnswerCorrect</sender>
   <signal>toggled(bool)</signal>
   <receiver>eAnswerCorrect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hasAnswerIncorrect</sender>
   <signal>toggled(bool)</signal>
   <receiver>eAnswerIncorrect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hasNoShuffle</sender>
   <signal>toggled(bool)</signal>
   <receiver>eNoShuffle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hasQuestionFinish</sender>
   <signal>toggled(bool)</signal>
   <receiver>eQuestionFinish</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hasQuestionStart</sender>
   <signal>toggled(bool)</signal>
   <receiver>eQuestionStart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
